### DISCUSSION TITLE
QTerminal preferences dialog tidying

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2327] [Ideas]
Hi all. Posting this to hopefully generate ideas on how to better organize the QTerminal preferences dialog. I _should_ be able to make a PR with the wanted changes.

#### Current status:
- Appearance tab has no group boxes.
- Behavior tab has a non-descriptive Behavior group box.
- Shortcuts/Dropdown/Bookmarks are fine in my estimation.

![screen1](https://user-images.githubusercontent.com/96850319/201493162-6a175ee7-0b4c-4cba-b802-809971e9ea1e.png)
![screen2](https://user-images.githubusercontent.com/96850319/201493164-997c4c82-0ff4-474d-bb3c-73ff47c71252.png)

**Comment by tsujan** [comment]
Yes, QTerminal's Preferences dialog needs some love. It does the job correctly, but finding its relevant options can be difficult.
